core/templates/components/ratings/rating-computation/rating-computation.service.ts

```diff
@@ -49,10 +49,6 @@ export class RatingComputationService {
         totalNumber += ratingFrequencies[value];
       }
 
-      if (totalNumber === 0) {
-        return undefined;
-      }
-
```

Comment:
Why was this removed? We don't want to remove code for the sake of increasing test coverage.

Reply:
Hi @kevinlee12, this was removed because this branch of code is unreachable. In the first if statement evaluation in RatingComputationService, we check to see if totalNumber is greater or equal to the number of acceptable ratings (which is 1).  We then reach this branch of code where it only executes if totalNumber is greater than or equal to 1 while equaling zero, which would never be possible. Thus, this piece of code was removed since it would never be executed.

Reply:
ah, okay.